UI-Session-4-Assignment/index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Home</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <!-- header for all web pages of this site contain logo-image ,menu and social links of facebook instagram and google plus-->
+    <header id="header">
+        <!--company logo-->
+        <img class="logo" src="images\logo.png" alt="Logo image">
+        <!-- menu unordered list-->
+        <ul class="menu">
+            <li> <a href="">Home</a> </li>
+            <li> <a href="">Features</a> </li>
+            <li> <a href="">Review</a> </li>
+            <li> <a href="">Download</a> </li>
+        </ul>
+        <!-- social links of company-->
+        <div class="social-links">
+            <a href=""> <i class="fab fa-facebook-f"></i> </a>
+            <a href=""> <i class="fab fa-instagram"></i> </a>
+            <a href=""> <i class="fab fa-google-plus-g"></i> </a>
+        </div>
+    </header>
+
+    <!-- cover section containing Download and feature option -->
+    <section id="cover-section">
+        <div>
+            <h1>Get the most fun weather app</h1>
+            <a href="" id="download_button">Download</a>
```

Comment:
It should have a red color same as layout.

Reply:
Done